Formula/h/hubble.rb

```diff
@@ -5,6 +5,14 @@ class Hubble < Formula
   sha256 "c4c2e5ed223c5c4d8777f1da9b44623373240d940365910adb501636a07faecb"
   license "Apache-2.0"
 
+  # Upstream creates stable version tags (e.g., `v1.2.3`) before a release but
+  # the version isn't considered to be released until a corresponding release
+  # is created on GitHub, so it's necessary to use the `GithubLatest` strategy.
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
```

The "isn't considered to be released until a corresponding release is created on GitHub" comment is for projects that explicitly use GitHub releases to indicate that a version is released (i.e., confirmed by upstream). Upstream hasn't commented on the related issue yet, so the "notable gap" comment is arguably more appropriate here (it's more of an indication of how we handle version releases in homebrew/core).